docs/ops/normalization/NormalizeL2_1.md

```diff
@@ -4,7 +4,22 @@
 
 **Category**: *Normalization*
 
-**Short description**: *NormalizeL2* operation performs L2 normalization of the 1st input tensor in slices specified by the 2nd input.
+**Short description**: *NormalizeL2* operation performs L2 normalization of the first input tensor in slices specified by the second input.
+
+**Detailed Description**
+
+Each element in the output is the result of division of corresponding element from the `data` input tensor by the result of L2 reduction along dimensions specified by the `axes` input:
```

Comment:
the result of division of corresponding element from the \`data\` input tensor by the result of L2 reduction ---> the result of dividing the corresponding element of \`data\` input by the result of L2 reduction

Reply:
Done